build/Targets/GetProjectsReferencingProjectJsonFiles.targets

```diff
@@ -1,18 +1,20 @@
-﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <UsingTask TaskName="ProjectReferencesTask" AssemblyFile="$(NuGetTasksAssemblyPath)" Condition=" $(MSBuildThisFileExtension) == '.xproj' " />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="ProjectReferencesTask" AssemblyFile="$(NuGetTasksAssemblyPath)" Condition="'$(MSBuildThisFileExtension)' == '.xproj'" />
   <Target Name="_NuGet_GetProjectsReferencingProjectJsonInternal"
           DependsOnTargets="_SplitProjectReferencesByFileExistence"
           Returns="@(_ProjectReferencingProjectJsonFile)">
 
     <!-- Process xprojs with an external task to find xproj -> xproj references -->
-    <ProjectReferencesTask ProjectFile="$(MsBuildProjectFullPath)" Condition=" $(MSBuildThisFileExtension) == '.xproj' " >
+    <ProjectReferencesTask ProjectFile="$(MsBuildProjectFullPath)" Condition="'$(MSBuildThisFileExtension)' == '.xproj'">
```

Comment:
@jaredpar, the surrounding quotes where missing here (and another below). Not sure what this may have impacted.